google.py

```diff
@@ -243,6 +243,10 @@ def status(self, text, match, conlist):
                 self.logger.error(e)
         for si in self.slack_instances:
             output = dict(self.slack_payload)
+            if si.get('players'):
+                for country,name in si.get('players').items():
+                    newtext = "%s (%s)"%(country,name) 
```

Comment:
should be single quoted and using the same format strings as the rest of the repo (ie: f'{country} {name}'

Reply:
and "country,name" => "country, name" (add space) to follow flake8 standard